action.yaml

```diff
@@ -25,6 +25,9 @@ inputs:
     required: false
     description: Limit the number of concurrent operations during plan/apply
     default: '10'
+  terraform_targets:
+    required: false
+    description: 'A comma separated string containing targets that should be passed to terraform. For example: foo,bar["one"],bar["two"]'
```

Comment:
Considered using a JSON string, like with `terraform_variables` below. This seems cleaner to me.